jest.config.cjs

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const nextJest = require("next/jest");
+/**
+ * Setting up Jest
+ * https://nextjs.org/docs/testing#setting-up-jest-with-the-rust-compiler
+ *  */
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: "./",
+});
+
+// Add any custom config to be passed to Jest
+/** @type {import('jest').Config} */
+const customJestConfig = {
+  // Add more setup options before each test is run
+  // setupFilesAfterEnv: ['<rootDir>/jest.setup.js'],
+  transformIgnorePatterns: ["node_modules/(?!(react-markdown))"],
+
+  setupFilesAfterEnv: ["<rootDir>/setupTests.js"],
+  testEnvironment: "jest-environment-jsdom",
+
+  moduleNameMapper: {
+    "~/(.*)$": "<rootDir>/src/$1",
+    "next/router": "<rootDir>/__mocks__/next/router.js",
+    "react-markdown": "<rootDir>/node_modules/react-markdown/react-markdown.min.js",
+  },
```

Comment:
what does this do? map to a different module during testing?

Reply:
There's a common issue with react-markdown and Jest where it has some issues compiling ESM to cjs I think, so this bypasses that by mapping to the minified file w/o ESM.